src/testUtils/java/uk/gov/hmcts/reform/waworkflowapi/SpringBootFunctionalBaseTest.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.waworkflowapi;
+
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("functional")
+public abstract class SpringBootFunctionalBaseTest {
```

Comment:
+1 This is nice too

Reply:
I think we have lost the Serenity reports if we do this haven't we?

Reply:
Thanks for spotting I've changed it to use the SpringIntegrationSerenityRunner seems to be generating the reports now :)